crates/ruff_linter/src/rules/refurb/snapshots/ruff_linter__rules__refurb__tests__preview__FURB169_FURB169.py.snap

```diff
@@ -0,0 +1,352 @@
+22 22 | 
+23 23 | type(foo) != type(None)
+24 24 | 
+
+FURB169.py:23:1: FURB169 [*] When checking against `None`, use `is not` instead of comparison with `type(None)`
+   |
+21 | type(None) == type(None)
+22 |
+23 | type(foo) != type(None)
+   | ^^^^^^^^^^^^^^^^^^^^^^^ FURB169
+24 |
+25 | type(None) != type(foo)
+   |
+   = help: Replace with `is not None`
+
+ℹ Safe fix
+20 20 | 
+21 21 | type(None) == type(None)
+22 22 | 
+23    |-type(foo) != type(None)
+   23 |+foo is not None
+24 24 | 
+25 25 | type(None) != type(foo)
+26 26 | 
+
+FURB169.py:25:1: FURB169 [*] When checking against `None`, use `is not` instead of comparison with `type(None)`
+   |
+23 | type(foo) != type(None)
+24 |
+25 | type(None) != type(foo)
+   | ^^^^^^^^^^^^^^^^^^^^^^^ FURB169
+26 |
+27 | type(None) != type(None)
+   |
+   = help: Replace with `is not None`
+
+ℹ Safe fix
+22 22 | 
+23 23 | type(foo) != type(None)
+24 24 | 
+25    |-type(None) != type(foo)
+   25 |+foo is not None
+26 26 | 
+27 27 | type(None) != type(None)
+28 28 | 
+
+FURB169.py:27:1: FURB169 [*] When checking against `None`, use `is not` instead of comparison with `type(None)`
+   |
+25 | type(None) != type(foo)
+26 |
+27 | type(None) != type(None)
+   | ^^^^^^^^^^^^^^^^^^^^^^^^ FURB169
+28 |
+29 | type(a.b) is type(None)
+   |
+   = help: Replace with `is not None`
+
+ℹ Safe fix
+24 24 | 
+25 25 | type(None) != type(foo)
+26 26 | 
+27    |-type(None) != type(None)
+   27 |+None is not None
+28 28 | 
+29 29 | type(a.b) is type(None)
+30 30 | 
+
+FURB169.py:29:1: FURB169 [*] When checking against `None`, use `is` instead of comparison with `type(None)`
+   |
+27 | type(None) != type(None)
+28 |
+29 | type(a.b) is type(None)
+   | ^^^^^^^^^^^^^^^^^^^^^^^ FURB169
+30 |
+31 | type(
+   |
+   = help: Replace with `is None`
+
+ℹ Safe fix
+26 26 | 
+27 27 | type(None) != type(None)
+28 28 | 
+29    |-type(a.b) is type(None)
+   29 |+a.b is None
+30 30 | 
+31 31 | type(
+32 32 |     a(
+
+FURB169.py:31:1: FURB169 [*] When checking against `None`, use `is not` instead of comparison with `type(None)`
+   |
+29 |   type(a.b) is type(None)
+30 |
+31 | / type(
+32 | |     a(
+33 | |         # Comment
+34 | |     )
+35 | | ) != type(None)
+   | |_______________^ FURB169
+36 |
+37 |   type(
+   |
+   = help: Replace with `is not None`
+
+ℹ Unsafe fix
+28 28 | 
+29 29 | type(a.b) is type(None)
+30 30 | 
+31    |-type(
+32    |-    a(
+33    |-        # Comment
+34    |-    )
+35    |-) != type(None)
+   31 |+a() is not None
+36 32 | 
+37 33 | type(
+38 34 |     a := 1
+
+FURB169.py:37:1: FURB169 [*] When checking against `None`, use `is` instead of comparison with `type(None)`
+   |
+35 |   ) != type(None)
+36 |
+37 | / type(
+38 | |     a := 1
+39 | | ) == type(None)
+   | |_______________^ FURB169
+40 |
+41 |   type(
+   |
+   = help: Replace with `is None`
+
+ℹ Safe fix
+34 34 |     )
+35 35 | ) != type(None)
+36 36 | 
+37    |-type(
+38    |-    a := 1
+39    |-) == type(None)
+   37 |+(a := 1) is None
```

Comment:
Is it correct that we remove the `type` around the `a := 1` here? same for the next diagnostic. Shouldn't this be `type(a := 1) is None`?

Reply:
oh dang sorry, I got confused what the rule does.